plugins/dfu/README.md

```diff
@@ -32,6 +32,42 @@ the device again re-enumerates back to the runtime mode.
 For this reason the `REPLUG_MATCH_GUID` internal device flag is used so that
 the bootloader and runtime modes are treated as the same device.
 
+## Implementation Notes
```

Comment:
As a general note; wouldn't this make more sense to end up on fwupd.github.io?  I know we don't really do a good job segementing documentation between README.md and what ends up there, but I think this is the type of thing we would want to end up there.  Maybe we even want to split up the `README.md` across all plugins to automagically build a table of information on fwupd.github.io.

In an ideal world I think someone should only need to go there to find out info and it would link out to other documentation at fwupd.org and readthedocs etc.

Reply:
Agreed, but in a different pull request :) https://github.com/fwupd/fwupd/pull/5393